src/components/AudioPlayer.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, useImperativeHandle, useRef } from "react";
```

Comment:
Removed from canvas. I don't think we use this much, but when we do, there is no point to wait for the canvas to show up